WordPressShared/Core/Utility/DisplayableImageHelper.m

```diff
@@ -10,6 +9,21 @@
 
 @implementation DisplayableImageHelper
 
++ (NSInteger)widthOfAttachment:(NSDictionary *)attachment {
+    NSInteger result = 0;
+    id obj = [attachment objectForKey:AttachmentsDictionaryKeyWidth];
+    if ([obj isKindOfClass:NSNumber.class]) {
+        NSNumber *number = (NSNumber *)obj;
+        result = [number integerValue];
+    } else if ([obj isKindOfClass:NSString.class]) {
+        NSNumberFormatter *numberFormatter = [[NSNumberFormatter alloc] init];
+        numberFormatter.locale = [NSLocale localeWithLocaleIdentifier:@"en_US_POSIX"];
+        NSNumber *number= [numberFormatter numberFromString:(NSString *)obj];
+        result = [number integerValue];
+    }
+    return result;
+}
```

Comment:
Nipticky suggestion! How about adding...

```
@interface NSDictionary ()
- (NSInteger)integerForKey:(NSString *)key;
@end
```

That way, if we needed this snippet anywhere else in the entire POD, we could just reuse it.